docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
       retries: 3
       start_period: 30s
     environment:
+      # Run "composer require symfony/orm-pack" to install and configure Doctrine ORM
+      DATABASE_URL: mysql://symfony:ChangeMe@db:3306/app?serverVersion=mariadb-10.5.6
```

Comment:
Now the variable can be in the override ? And maybe write it like `${DATABASE_URL:-mysql://symfony:ChangeMe@db:3306/app?serverVersion=mariadb-10.5.6}` ?